packages/crawler/src/storage/local-blob-store.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import Apify from 'apify';
+import { BlobStore } from './store-types';
+
+export class LocalBlobStore implements BlobStore {
+    constructor(
+        public readonly storeName: string,
+        private keyValueStore?: Apify.KeyValueStore,
+        private readonly apifyObj: typeof Apify = Apify,
+    ) {}
+
+    public async setValue(key: string, value: string | Object, options?: { contentType?: string }): Promise<void> {
+        await this.open().then(async () => {
```

Comment:
why do we need to use then() and await on promise chain?

Reply:
same like above